vipps-report-api-faq.md

```diff
@@ -23,34 +23,46 @@ For more common Vipps questions, see:
 
 * [Vipps API General FAQ](https://vippsas.github.io/vipps-developer-docs/docs/vipps-developers/faqs/)
 
-Document version: 0.0.6.
+Document version: 1.0.0.
 
 <!-- START_TOC -->
 
 ## Table of contents
 
 * [When will the API be available?](#when-will-the-api-be-available)
+* [What information can I get hold of?](#what-information-can-i-get-hold-of)
 * [Which API keys give access to the API?](#which-api-keys-give-access-to-the-api)
   * [The merchant's own API keys](#the-merchants-own-api-keys)
   * [Partner keys](#partner-keys)
   * [Specifying a accounting partner](#specifying-a-accounting-partner)
+  * [I keep getting an empty list in response when calling one of the endpoints](#i-keep-getting-an-empty-list-in-response-when-calling-one-of-the-endpoints)
 * [Questions?](#questions)
 
 <!-- END_TOC -->
 
 ## When will the API be available?
 
-We can not give a date, month or any kind of ETA yet.
+The Vipps Report API will be available from January 9 2023.
```

Comment:
Should we say something about it being Beta and that backwards-incompatible changes can still be made? Or do we handle that if it arises? Which it probably doesn't I guess... @cloveras